### DISCUSSION TITLE
Volume and brightness in actions

### DISCUSSION THREAD

**Discussion by 7OH** [luisbocanegra/plasma-panel-spacer-extended#28] [General]
Will be nice to see ability to change music volume and brightness for actions (mouse wheel).
Yes - we can change it with wheel on battery or volume icon, but panel is MORE bigger than one small icon.

**Comment by luisbocanegra** [comment]
Volume can be controlled with **kmix - increase/decrease_volume** and brightness can be controlled by **org_kde_powerdevil - Increase/Decrease Screen Brightness**

**Comment by 7OH** [comment]
> Volume can be controlled with **kmix - increase/decrease_volume** and brightness can be controlled by **org_kde_powerdevil - Increase/Decrease Screen Brightness**

If  You mean write it in custom comand then:
1. kmix needs to be installed
2. kmix increase_volume just start kmix window
3. org_kde_powerdevil do nothing - in terminal "Command not found"

**Comment by luisbocanegra** [comment]
> If You mean write it in custom comand then:

Sorry I wasn't clear, they're actions (shortcuts under the hood) that you can choose from the widget

>     1. kmix needs to be installed

Yeah, after installing kmix (or anything else that creates shortcuts, e.g. a KWin Script) they will show up on the widget:

![Screenshot_20240427_125412](https://github.com/luisbocanegra/plasma-panel-spacer-extended/assets/15076387/c3e52c84-10a5-4e74-ac4f-fe462bdc0bb6)

**Comment by ThomasSeeker** [comment]
Hey there!

I have the normal integrated Plasma-Applet "Volume" which works with shortcuts. But it does not show up in the widgets drop-down-menu. ... Unfortunately, because using "Custom Command" with xdotool with the shortcut reacts way slower, i.e. too slow.

I even installed kmix (and destroyed the integrated Plasma-Applet) ... but no entry in the widgets drop-down-menu either. 

What am I missing here?
